src/cmd/build.js

```diff
@@ -1,30 +1,31 @@
 /* @flow */
 import path from 'path';
-import minimatch from 'minimatch';
 import {createWriteStream} from 'fs';
-import streamToPromise from 'stream-to-promise';
+
+import minimatch from 'minimatch';
 import {fs} from 'mz';
+import streamToPromise from 'stream-to-promise';
 import parseJSON from 'parse-json';
 
 import defaultSourceWatcher from '../watcher';
 import {zipDir} from '../util/zip-dir';
 import getValidatedManifest, {getManifestId} from '../util/manifest';
 import {prepareArtifactsDir} from '../util/artifacts';
 import {createLogger} from '../util/logger';
-import {UsageError} from '../errors';
+import {UsageError} from '../errors'; //eslint-disable-line import/order
```

Comment:
This happens in a couple of other places as well. However, it works in `run.js`. I've been able to figure out why. I tried mimicking everything as is but still it didn't work.